node/sc/event_interface.go

```diff
@@ -0,0 +1,96 @@
+package sc
+
+import (
+	"math/big"
+
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	bridgecontract "github.com/klaytn/klaytn/contracts/bridge"
+)
+
+type IRequestValueTransferEvent interface {
+	Nonce() uint64
+	GetTokenType() uint8
+	GetFrom() common.Address
+	GetTo() common.Address
+	GetTokenAddress() common.Address
+	GetValueOrTokenId() *big.Int
+	GetRequestNonce() uint64
+	GetFee() *big.Int
+	GetExtraData() []byte
+
+	GetRaw() types.Log
+}
+
+//////////////////// type RequestValueTransferEvent struct ////////////////////
+// RequestValueTransferEvent from Bridge contract
+type RequestValueTransferEvent struct {
+	*bridgecontract.BridgeRequestValueTransfer
+}
+
+func (rEv RequestValueTransferEvent) Nonce() uint64 {
+	return rEv.RequestNonce
+}
```

Comment:
It has both `Nonce` and `GetRequestNonce` methods, and the two methods return the same value.

Reply:
This originated from the origin codebase. The origin codebase used jumbly two methods. I didn't change it because I experienced some of the guys holding my changes back about wrong semantics in origin even though the change is logically right.

Reply:
Okay. Let's consider this later when we need to refine the code